plugins/node/opentelemetry-instrumentation-aws-sdk/test/sqs.test.ts

```diff
@@ -361,6 +363,49 @@ describe('SQS', () => {
   });
 
   describe('hooks', () => {
+    it('sqsResponseHook add messaging attributes', async () => {
```

Comment:
nit: this test is specifically for attributes added during `sendMessage`. Might be worth mentioning in the test name

Reply:
Fixed in https://github.com/open-telemetry/opentelemetry-js-contrib/pull/968/commits/0f6155c856f52fcdde568b1ea682c84f167c74f0